tRecorderApi/htmlcov/api_migrations_0001_initial_py.html

```diff
@@ -150,11 +150,12 @@ <h2 class="stats">
 <p id="n81" class="pln"><a href="#n81">81</a></p>
```

Comment:
gitignoring runtime generated files is the quickest way to clean up your PRs and not make them as cumbersome.